public/app/features/explore/Logs/LogsTableWrap.tsx

```diff
@@ -278,6 +278,25 @@ export function LogsTableWrap(props: Props) {
 
   const styles = useStyles2(getStyles, height, sidebarWidth);
 
+  const onSortByChange = useCallback(
+    (sortBy: Array<{ displayName: string; desc?: boolean }>) => {
+      // Transform from Table format to URL format - only store the first sort column
+      if (sortBy.length > 0) {
+        // Defer the Redux dispatch to avoid updating ExploreActions during Table's render cycle
+        // Even though this is called from an event handler, the synchronous Redux dispatch
+        // can cause ExploreActions (which subscribes to panes state) to re-render while
+        // Table is still rendering, triggering the React warning
```

Comment:
It's a bit unfortunate that we need to do this. Probably related, but every time I use the Table I feel that it re-renders too much with any action.